src/engine/SCons/compat/__init__.py

```diff
@@ -71,8 +71,8 @@ def import_as(module, name):
     Imports the specified module (from our local directory) as the
     specified name, returning the loaded module object.
     """
-    dir = os.path.split(__file__)[0]
-    return imp.load_module(name, *imp.find_module(module, [dir]))
+    sys.modules[name] = importlib.import_module(module)
```

Comment:
`import_as()` seems to be unused, maybe we can just drop it?

Reply:
done